paper.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "PyMedPhys: A community effort to develop an open standard library for Medical Physics in Python"
+tags:
+  - Python
+  - Medical Physics
+  - Radiation Therapy
+  - Diagnostic Imaging
+  - DICOM
+authors:
+  - name: Simon Biggs
+    affiliation: 1
+  - name: Stuart Swerdloff
+    affiliation: 2
+  - name: Matthew Jennings
+    affiliation: 4
+  - name: Phillip Chlap
+    affiliation: "5, 6"
+  - name: Derek Lane
+    affiliation: 3
+  - name: Jacob Rembish
+    affiliation: 7
+  - name: Jacob McAloney
+    affiliation: 8
+  - name: Paul King
+    affiliation: 9
+  - name: Rafael Ayala
+    affiliation: 10
+  - name: Matthew Sobolewski
+    affiliation: "8, 11"
+affiliations:
+  - name: Radiotherapy AI, Australia
+    index: 1
+  - name: ELEKTA Pty Ltd, New Zealand
+    index: 2
+  - name: Elekta AB
+    index: 3
+  - name: Royal Adelaide Hospital, Australia
+    index: 4
+  - name: University of New South Wales, Australia
+    index: 5
+  - name: Ingham Institute, Australia
+    index: 6
+  - name: NYU Langone Health, USA
+    index: 7
+  - name: Riverina Cancer Care Centre, Australia
+    index: 8
+  - name: Painless Skin Cancer Treatment, Mississippi, USA
+    index: 9
+  - name: Hospital G.U. Gregorio Marañón, Spain
+    index: 10
+  - name: CancerCare Partners, Australia
+    index: 11
+date: 10 February 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+PyMedPhys is an open-source medical physics Python library built by a community
+that values and prioritizes code sharing, review, improvement, and peer
+development. The overall goal of PyMedPhys is to provide medical physicists
+around the world with a free, open-source library of Python applications to
+simplify and enhance both their research and clinical work. It is inspired by
+the collaborative work of our physics peers in astronomy and the Astropy
+Project [@astropy].
+
+# Statement of need
+
+As the treatment delivery methods used in radiation oncology continue to
+improve, the complexity associated with ensuring precise and accurate delivery
+continues to increase. In response, many medical physicists are turning toward
```

Comment:
Suggest a reword that at least acknowledges non-radiotherapy medical physics. Also suggest alternative to "delivery methods", since this might imply innovation beam _delivery_ techniques only when there have been significant innovations relevant to PyMedPhys in simulation, planning, guidance, etc.

Perhaps the paragraph could be replaced with something like:

_"Medical radiation applications are subject to fast-paced technological advancements. This is particularly true in the field of radiation oncology, where the implementation of increasingly sophisticated technologies requires increasingly complex processes to maintain the improving standards of care. To address this challenge, medical physicists are increasingly developing in-house software tools to improve the quality, safety and efficiency of clinical tasks. However, in many cases, such development is limited to the physicist's own clinic. Consequently, many similar tools that could otherwise be shared are instead "reinvented" on a routine basis. Moreover, individual clinics often lack the personnel resources to incorporate simple good development practices such as peer-review."_

Just a draft that could be tweaked. Noticed I've removed the AAPM comment. That could be reintroduced, but I would suggest that such a claim needs a reference to be included.